deb/sphinx/DEBIAN/control

```diff
@@ -1,5 +1,5 @@
 Package: sphinx
-Version: 0.3.1
+Version: 0.4.1
```

Comment:
Because I messed up. It wasn't going to break anything so I left it.
On Jun 19, 2015 1:34 PM, "Ryan Endacott" <notifications@github.com> wrote:

> In deb/sphinx/DEBIAN/control
> <https://github.com/Clever/sphinx/pull/49#discussion_r32866868>:
>
> > @@ -1,5 +1,5 @@
> >  Package: sphinx
> > -Version: 0.3.1
> > +Version: 0.4.1
>
> I realize this is already committed, but I'm curious why this is 0.4.1
> instead of 0.4.0?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Clever/sphinx/pull/49/files#r32866868>.
>